### ISSUE TITLE
Font sizes

### ISSUE THREAD

**Issue by jsixface** [quillpad/quillpad#273]
Right now, in the settings, there is a "Text editor/view font size" setting. This is in numbers which doesn't make sense to a layman user. Remove the numerical values in the list of options and convert them to a list of sizes like small, medium, large and extra large. 

Migrate the existing users' settings from numbers if they have selected other than default, to one of the named sizes.

**Comment by clach04** [comment]
I disagree, [point](https://en.m.wikipedia.org/wiki/Point_(typography)) is a standard unit.

**Comment by LIGMATV** [comment]
It's more like a downgrade than a feature request.
